source/includes/siteurls-config/_10-show.md.erb

```diff
@@ -0,0 +1,39 @@
+## Get SiteUrls Config
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/config/server/site_urls' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.siteurls_config %>'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.siteurls_config %>; charset=utf-8
+```
+
+```json
+{
+  "_links": {
+    "doc": {
+      "href": "https://api.gocd.org/19.11.0/#create-site-url"
```

Comment:
Should this be `create-site-url` or `get-siteurls-config`?

Reply:
I checked the code, it's documented as `create-site-url` since the API returns the same. Ideally I think it should link to top level SiteUrls docs that is `siteurls-config`. If you are changing the the docs, we will have to change in the code as well.

Reply:
Yes. I agree with your point , will update it.